packages/one-app-runner/__tests__/src/startApp.spec.js

```diff
@@ -77,166 +73,177 @@ describe('startApp', () => {
-    const mockSpawn = require('mock-spawn')();
-
+    try {
+      await startApp({
+        moduleMapUrl: 'https://example.com/module-map.json', rootModuleName: 'frank-lloyd-root', appDockerImage: 'one-app:5.0.0', modulesToServe: ['/path/to/module-a'], createDockerNetwork: true, dockerNetworkToJoin: 'one-test-environment-1234',
+      });
+    } catch (error) {
+      expect(error).toMatchSnapshot('create network calls');
+    }
+  });
+
+  it('uses host instead of localhost when the useHost flag is passed', async () => {
+    expect.assertions(1);
+    const mockSpawn = makeMockSpawn();
     childProcess.spawn.mockImplementationOnce(mockSpawn);
-    startApp({
+    await startApp({
       moduleMapUrl: 'https://example.com/module-map.json', rootModuleName: 'frank-lloyd-root', appDockerImage: 'one-app:5.0.0', modulesToServe: ['/path/to/module-a'], useHost: true,
     });
     expect(mockSpawn.calls[0].command).toMatchSnapshot();
   });
 
-  it('bypasses docker pull when the offline flag is passed', () => {
-    const mockSpawn = require('mock-spawn')();
-
+  it('bypasses docker pull when the offline flag is passed', async () => {
+    expect.assertions(1);
+    const mockSpawn = makeMockSpawn();
     childProcess.spawn.mockImplementationOnce(mockSpawn);
-    startApp({
+    await startApp({
       moduleMapUrl: 'https://example.com/module-map.json', rootModuleName: 'frank-lloyd-root', appDockerImage: 'one-app:5.0.0', modulesToServe: ['/path/to/module-a'], offline: true,
     });
     expect(mockSpawn.calls[0].command).toMatchSnapshot();
   });
 
-  it('Passes the container name to the docker --name flag', () => {
-    const mockSpawn = require('mock-spawn')();
-
+  it('Passes the container name to the docker --name flag', async () => {
+    expect.assertions(1);
+    const mockSpawn = makeMockSpawn();
     childProcess.spawn.mockImplementationOnce(mockSpawn);
-    startApp({
+    await startApp({
       moduleMapUrl: 'https://example.com/module-map.json', rootModuleName: 'frank-lloyd-root', appDockerImage: 'one-app:5.0.0', modulesToServe: ['/path/to/module-a'], containerName: 'one-app-at-test',
     });
     expect(mockSpawn.calls[0].command).toMatchSnapshot();
   });
 
-  it('outputs all logs from docker pull and docker run to a file if output file arg is given', () => {
-    const mockSpawn = require('mock-spawn')();
-    const mockStdout = 'hello world (stdout)';
-    const mockStderr = 'hello erroneous world (stderr)';
-    mockSpawn.setDefault(mockSpawn.simple(0, mockStdout, mockStderr));
+  it('outputs all logs from docker pull and docker run to a file if output file arg is given', async () => {
+    expect.assertions(5);
+    const mockProcess = { on: jest.fn(), stdout: { pipe: jest.fn() }, stderr: { pipe: jest.fn() } };
+    const mockSpawn = jest.fn(() => mockProcess);
     childProcess.spawn.mockImplementationOnce(mockSpawn);
-
-    startApp({
+    createWriteStreamSpy.mockReturnValueOnce('stream');
```

Comment:
This gets rid of the need for that file that was previously being deleted in `afterAll`